src/commands/deploy.ts

```diff
@@ -232,10 +236,25 @@ export default class Deploy extends DeployCommand {
     const account = await AccountUtils.getAccount(this.app, flags.account);
     const environment = await EnvironmentUtils.getEnvironment(this.app.api, account, flags.environment);
 
-    const deployment_dtos = [];
+    const component_names: string[] = [];
     for (const component of components) {
+      if (fs.existsSync(component)) {
+        const register = new ComponentRegister([component, '-a', account.name, '-e', environment.name], this.config);
+        register.app = this.app;
+        await register.run();
+        const component_spec = buildSpecFromPath(component);
+        component_names.push(`${account.name}/${component_spec.name}:${ComponentRegister.getTagFromFlags({ environment: environment.name })}`);
```

Comment:
Remove the account name prefix. Since sometimes the account name is part of the name for older customers.

```ts
component_names.push(`${component_spec.name}:${ComponentRegister.getTagFromFlags({ environment: environment.name })}`);
```